store/src/java/com/zimbra/cs/lmtpserver/ExternalEmailWarning.java

```diff
@@ -173,26 +185,38 @@ public String getUpdatedContent(String content) {
         if (content != null) {
             final Matcher ctTextPlainMatcher = CONTENT_TYPE_TEXT_PLAIN_PATTERN.matcher(content);
             if (ctTextPlainMatcher.find()) {
+                String textPlainHeader = ctTextPlainMatcher.group();
                 final int end = ctTextPlainMatcher.end();
-                final StringBuilder sb = new StringBuilder();
-                sb.append(content.substring(0, end));
-                if (getTextPlainWarning() != null)
-                    sb.append(getTextPlainWarning());
-                sb.append(content.substring(end));
-                content = sb.toString();
+                if (textPlainHeader.contains("Content-Transfer-Encoding: base64")) {
```

Comment:
Create constant for "Content-Transfer-Encoding: base64"

Reply:
created CONTENT_TRANSFER_BASE64_ENCODING constant